File: ios/EventPropertiesOutputExtensions.swift
import Foundation

extension EventPropertiesOutput {
  static func boolean(_ value: Bool) -> EventPropertiesOutput {
    return .first(value)
  }

  static func string(_ value: String) -> EventPropertiesOutput {
    return .third(value)
  }

  static func number(_ value: Double) -> EventPropertiesOutput {
    return .second(value)
  }
}

File: android/src/main/java/com/margelo/nitro/rive/EventPropertiesOutputExtensions.kt
package com.margelo.nitro.rive

object EventPropertiesOutputExtensions {
  fun boolean(value: Boolean): EventPropertiesOutput =
    EventPropertiesOutput.First(value)

  fun number(value: Double): EventPropertiesOutput =
    EventPropertiesOutput.Second(value)

  fun string(value: String): EventPropertiesOutput =
    EventPropertiesOutput.Third(value)
}
